backend/Dockerfile

```diff
@@ -25,6 +26,10 @@ RUN if [ "$use_remote_build" = "true" ]; then \
 
 # Compile
 FROM python:3.5 as compiler
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y -q default-jdk python3-setuptools python3-dev
+RUN wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py
+RUN python3 -m pip install tfx==0.15.0
```

Comment:
Maybe we should just start with the tfx image?

Reply:
Good point :) But unfortunately no matter what, compile this sample here will need ad hoc logic because we cannot call `dsl-compile` against it.

I am skipping the compilation here and will use tfx image as the base image for sample test.

Reply:
> we cannot call `dsl-compile` against it.

We can just run the sample .py files as they're self-compiling.

Reply:
There might be a mismatch between names/extensions produced by the pipeline compilation and the extensions in the backend sample pipeline config.

Reply:
Thanks! I've been stuck in this trap for a while... Will give it a shot.

Reply:
The reason turns out to be Dockerfile compiled parameterized_tfx_oss.yaml under sample dir instead of under paramterized_tfx_oss dir due to the hard-coded output file name.